pandas/core/generic.py

```diff
@@ -590,7 +590,7 @@ def swapaxes(self: FrameOrSeries, axis1, axis2, copy=True) -> FrameOrSeries:
         if copy:
             new_values = new_values.copy()
 
-        return self._constructor(new_values, *new_axes).__finalize__(
+        return self._constructor(new_values, *new_axes).__finalize__(  # type: ignore
```

Comment:
hmm, can you add the mypy error as a comment.

IIUC there were issues with deprecating this method, xref #26654.

For series, it's a no-op. maybe now that Panel is deprecated, we could have logic in DataFrame and Series instead of NDFrame and simplify.

Reply:
Yeah, I remember that. The problem was that ``np.swapaxes(df, 0, 1)``, ``np.swapaxes(series, 0, 0)`` works in master and are tested for. It's not unreasonable that it works, but users should really use ``transpose``/``T`` instead.

Maybe add ``swapaxes`` to ``_deprecations`` to get it out of the tab-list?